react/src/App.js

```diff
@@ -10,7 +10,7 @@ function App() {
 
   return (
       <div className="App">
-      <MoonPhase date={today}/>
```

Comment:
Remove date argument for MoonPhase since MoonPhase applies current time on its own